lib/tracker.ex

```diff
@@ -19,6 +19,11 @@ defmodule Tracker do
     end)
   end
 
+  def complete? do
+    {total, completed} = Agent.get(__MODULE__, &(&1))
```

Comment:
There is a bit of duplication here and in `get/0`. I might suggest we refactor `get/0` to return the data itself and update `summarize/0` to be based on `get/0` rather than the other way around.